index.js

```diff
@@ -86,11 +86,11 @@ app.use(
         accessToken,
         userID,
         defaultTimezone,
-        ...perTypeLoaders(),
+        ...createLoaders(accessToken, userID),
```

Comment:
Seems like this file wasn’t run through prettier yet and thus has a bunch of style changes. This is the only real change.